thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/DetectionUtils.java

```diff
@@ -55,8 +56,8 @@
   // TODO anomaly should support multimap
   public static DimensionMap toFilterMap(Multimap<String, String> filters) {
     DimensionMap map = new DimensionMap();
-    for (Map.Entry<String, String> entry : filters.entries()) {
-      map.put(entry.getKey(), entry.getValue());
+    for (Map.Entry<String, Collection<String>> entry: filters.asMap().entrySet()){
+      map.put(entry.getKey(), String.join(", ", entry.getValue()));
```

Comment:
Any reason use ", " instead of ","?

Reply:
This dimension map is only used for presenting. Using ", " looks better than ",".